src/less/utility/base/utility.less

```diff
@@ -63,6 +63,21 @@
     }
 }
 
+//greys out space in the li not taken by the img, in order to match height to other img
+// .image-treatment {
```

Comment:
I still have a question about making this a utility class - I tried but it wasn't quite panning out